apps/finance/test/mocks/FinanceMock.sol

```diff
@@ -3,8 +3,12 @@ pragma solidity 0.4.18;
 import "../../contracts/Finance.sol";
 
 contract FinanceMock is Finance {
-    uint _mockTime = now;
+    uint256 mockTime = now;
+    uint mockMaxPeriodTransitions = MAX_UINT;
```

Comment:
(Nitpicking, or missing something) Why are you changing `mockTime` to `uint256` but using `uint` for `mockMaxPeriodTransitions`?

Reply:
Ohh, good point!